website/src/controllers/groups.js

```diff
@@ -473,14 +473,15 @@ api.join = function(req, res, next) {
   if(!isUserInvited) return res.json(401, {err: "Can't join a group you're not invited to."});
 
   if (!_.contains(group.members, user._id)){
+
     if (group.members.length === 0) {
       group.leader = user._id;
     }
 
     group.members.push(user._id);
-
     if (group.invites.length > 0) {
      group.invites.splice(_.indexOf(group.invites, user._id), 1);
+     console.log(group.invites);
```

Comment:
This is the section giving me 'error: VersionError: No matching document found.' .